cmd/util/ledger/reporters/account_reporter.go

```diff
@@ -139,7 +139,10 @@ type balanceProcessor struct {
 
 func NewBalanceReporter(chain flow.Chain, view state.View) *balanceProcessor {
 	vm := fvm.NewVirtualMachine(fvm.NewInterpreterRuntime())
-	ctx := fvm.NewContext(zerolog.Nop(), fvm.WithChain(chain))
+	ctx := fvm.NewContext(zerolog.Nop(),
+		fvm.WithChain(chain),
+		// otherwise the memory limit gets overridden
+		fvm.WithAllowContextOverrideByExecutionState(false))
```

Comment:
can you send out a pr for the master branch as well, using the WithMemoryAndInteractionLimitsDisabled option instead WithAllowContextOverrideByExecutionState?

Reply:
https://github.com/onflow/flow-go/pull/3105